addon/components/truncate-multiline.js

```diff
@@ -148,7 +153,7 @@ export default Ember.Component.extend(ResizeHandlerMixin, {
   didRender() {
     this._super(...arguments);
     if (!this.get('_didTruncate') && this.get('_truncate')) {
-      Ember.run.scheduleOnce('afterRender', this, this._doTruncation);
+      scheduleOnce('afterRender', this, this._doTruncation);
```

Comment:
I wonder if we should consider using ember-app-scheduler instead for this? If we're trying to defer this until after the paint phase, this may be more accurate than relying on `afterRender`. Thoughts?

Reply:
I am not familiar with this code, but will take a stab in next PR